modules/mysql/testdata/mariadb/v5.5.46/process_list.txt

```diff
@@ -0,0 +1,6 @@
+*************************** 1. row ***************************
+TIME: 1
+USER: root
+*************************** 2. row ***************************
+TIME: 10
+USER: system user
```

Comment:
```suggestion
*************************** 1. row ***************************
TIME: 1
USER: root
*************************** 2. row ***************************
TIME: 10
USER: system user
*************************** 3. row ***************************
TIME: 5
USER: system user
```
Let's add another row so we can be sure we really find max TIME and not the last one